test/verify/check-machines-dbus

```diff
@@ -329,6 +329,13 @@ class TestMachinesDBus(machineslib.TestMachines):
         b.wait_present("li.active #vm-VmNotInstalled-consoles")
         b.click("#vm-VmNotInstalled-off-caret")
         b.click("#vm-VmNotInstalled-forceOff")
+        # https://bugzilla.redhat.com/show_bug.cgi?id=1818089
+        # After shutting it off virt-install will restart the domain and exit because of the above bug
+        # We need to wait till it's restarted and shut if off again in order ot edit the offline VM configuration
+        logfile = "/var/log/libvirt/qemu/VmNotInstalled.log"
+        wait(lambda: self.machine.execute("cat {0}".format(logfile)).count('starting up libvirt'), delay=3)
```

Comment:
count() is certainly a curious way to check if something contains a substring, but ok :) `lambda: "starting up libvirt" in execute(...)` seems a little more standard Python.

Reply:
I wanted to check that it appears '2' times, thus the count here - I should not work in weird Vm installation scripts when I am sleepy.